doc/usage.md

```diff
@@ -4,10 +4,10 @@ On Linux after building, the following executables are available in the project
 * picohttp_ct: runs various HTTP tests
 * picoquic_ct: runs various QUIC tests
 	* with no arguments, runs all tests.
-* picoquicdemo: QUIC/HTTP demo client and server
+* picoquicdemo: QUIC/HTTP demo client and server for HTTP/3, HTTP/0.9, QUIC performance tests and Siduck(simple ping-pong)
 * picoquic_sample: QUIC/HTTP demo client and server. See sample/README.md for detailed usage
 
-## HTTP CLI
+## HTTP Client and Server Usage
 * client: ./picoquicdemo servername serverportnumber HTTPpath
```

Comment:
There are a bunch of optional arguments that can be used by both client and server. You probably don't want to detail them all, but for the server it is important to understand setting encryption keys (-k) and setting certificate (-c). In any case, a pointer to using "-h" to get the list of options would be helpful.

Reply:
yeah, I've been thinking about adding some arguments, but way too many. That is why at the top of the file, I wrote: "All support "-h" argument to get the detailed usage". But I guess -k and -c are useful to recall explicitly.

Reply:
added wiki links in latest commit